core/modules/layout/plugins/renderers/layout_renderer_editor.inc

```diff
@@ -203,15 +203,19 @@ class LayoutRendererEditor extends LayoutRendererStandard {
    */
   protected function getBlockLinks($block) {
     $links = array();
-    $links['configure'] = array(
-      'title' => t('Configure'),
-      'href' => $this->getUrl('configure-block', $block->uuid),
-      'attributes' => array(
-        'class' => array('use-ajax'),
-        'data-dialog' => TRUE,
-        'data-dialog-options' => json_encode(array('dialogClass' => 'layout-dialog')),
-      ),
-    );
+
+    // Only render the "Configure" link if the block is not broken.
+    if (!$block->isBroken()) {
```

Comment:
Instead of a new method, we could simply check the class type and replace this with `if (!is_a($block, 'BlockBroken'))`

Reply:
nitpick :  `$block instanceof BlockBroken` is slightly faster (and I personally prefer this syntax, but it's a matter of personal taste).

Reply:
PR updated.